lib/openhab/core/items/semantics/enumerable.rb

```diff
@@ -72,15 +72,21 @@ def members
   # @!group Items State and Command Methods
 
   # Send a command to every item in the collection
-  # @return [self]
+  # @return [self, nil] nil when `ensure` is in effect and at least one of the items was already in the same state,
```

Comment:
so I would vote that it returns `self` if _any_ command was sent, and nil only if no commands were sent at all. I've just been through `Array` docs quite deeply, and learned that things like `Array#reject!`, `#select!`, `#flatten!`, and `uniq!` return `self` if anything is modified, and `nil` if nothing was modified.

Reply:
I've changed it so it would return nil only when _all_ members were already in the same state, otherwise return self.